Rakefile

```diff
@@ -27,60 +27,60 @@ require_relative "tasks/external_tests"
 require_relative "tasks/maintainers"
 require_relative "tasks/cbgb"
 
-ChefConfig::PackageTask.new(File.expand_path("..", __FILE__), "Chef") do |package|
+package_task = ChefConfig::PackageTask.new(File.expand_path("..", __FILE__), "Chef") do |package|
   package.component_paths = ["chef-config"]
   package.generate_version_class = true
 end
 
-task :pedant, :chef_zero_spec
-
-task :build_eventlog do
-  Dir.chdir "ext/win32-eventlog/" do
-    system "rake build"
-  end
-end
-
-task :register_eventlog do
-  Dir.chdir "ext/win32-eventlog/" do
-    system "rake register"
-  end
-end
-
-begin
-  require "chefstyle"
-  require "rubocop/rake_task"
-  RuboCop::RakeTask.new(:style) do |task|
-    task.options += ["--display-cop-names", "--no-color"]
-  end
-rescue LoadError
-  puts "chefstyle/rubocop is not available.  gem install chefstyle to do style checking."
-end
-
-begin
-  require "yard"
-  DOC_FILES = [ "README.rdoc", "LICENSE", "spec/tiny_server.rb", "lib/**/*.rb" ]
-  namespace :yard do
-    desc "Create YARD documentation"
-
-    YARD::Rake::YardocTask.new(:html) do |t|
-      t.files = DOC_FILES
-      t.options = ["--format", "html"]
-    end
-  end
-
-rescue LoadError
-  puts "yard is not available. (sudo) gem install yard to generate yard documentation."
-end
-
-begin
-  require "github_changelog_generator/task"
-
-  GitHubChangelogGenerator::RakeTask.new :changelog do |config|
-    config.future_release = Chef::VERSION
-    config.enhancement_labels = "enhancement,Enhancement,New Feature,Feature".split(",")
-    config.bug_labels = "bug,Bug,Improvement,Upstream Bug".split(",")
-    config.exclude_labels = "duplicate,question,invalid,wontfix,no_changelog,Exclude From Changelog,Question,Discussion".split(",")
-  end
-rescue LoadError
-  puts "github_changelog_generator is not available. gem install github_changelog_generator to generate changelogs"
-end
+# task :pedant, :chef_zero_spec
```

Comment:
I have no idea what you are talking about! Look, there aren't even any changes to the `Rakefile`!

(Thanks, updated :)